db/schema.rb

```diff
@@ -743,9 +752,9 @@
     t.string "declaration_type", null: false
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
-    t.index ["milestone_id", "schedule_id", "declaration_type"], name: "milestones_schedules_schedule_milestone_declaration_type", unique: true
+    t.index ["milestone_id", "schedule_id"], name: "index_schedule_milestones_on_milestone_id_and_schedule_id"
```

Comment:
if there's no associated migration then i think we should remove this

Reply:
Not sure where that came from. dropped and recreated local db